### ISSUE TITLE
Feature request: Option to jump right into tracking from reminder

### ISSUE THREAD

**Issue by kasbah** [SamAmco/track-and-graph#103]
Hey, love the app, thanks for making it. 

It would save me many clicks everyday if the reminder would jump right into the tracking when I press it. I'd like to go through all my tracking at that time and currently I have to select the group then to the + icon, then back to the next group etc... Some options to select which bits should be tracked for the reminder would be ideal.

**Comment by tevig** [comment]
perhaps I misunderstood, but here only part of my thought is the connection of the reminder with the tracker. it would be great to be able to group reminders into groups/folders.

**Comment by SamAmco** [comment]
yep putting reminders into groups is my preferred solution to this problem tbh

**Comment by SamAmco** [comment]
Rolling this out soon.
